README.md

```diff
@@ -144,13 +144,57 @@ With the newly added `fp8dqrow` scheme, we can bring down the inference latency
 
 </details>
 
+
 ### Trade-offs, trade-offs, and more trade-offs
 
 We know that the table included above is hard to parse. So, we wanted to include a couple of points that are worth noting. 
 
 * Select the quantization technique that gives you the best trade-off between memory and latency. 
 * A quantization technique may exhibit different optimal settings for a given batch size. For example, for a batch size of 4, `int8dq` gives best time without any QKV fusion. But for other batch sizes, that is not the case.
 
+The section below, drives this point home.
+
+#### Higher batch sizes like 16 and above
+
+This is how the top-5 latency looks like: 
+
+<details>
+<summary>Collapse table</summary>
+
+|    | ckpt_id                      |   batch_size | fuse   | compile   | compile_vae   | quantization   | sparsify   |   model_memory |   inference_memory |   time |
+|---:|:-----------------------------|-------------:|:-------|:----------|:--------------|:---------------|:-----------|---------------:|-------------------:|-------:|
+|  0 | black-forest-labs/FLUX.1-dev |           16 | False  | True      | True          | fp8dq          | False      |         20.356 |             52.704 | 45.004 |
+|  1 | black-forest-labs/FLUX.1-dev |           16 | False  | True      | True          | fp8dqrow       | False      |         20.368 |             52.715 | 45.521 |
+|  2 | black-forest-labs/FLUX.1-dev |           16 | True   | True      | False         | fp8dq          | False      |         22.363 |             52.464 | 45.614 |
+|  3 | black-forest-labs/FLUX.1-dev |           16 | False  | True      | False         | fp8dq          | False      |         20.356 |             50.458 | 45.865 |
+|  4 | black-forest-labs/FLUX.1-dev |           16 | False  | True      | False         | fp8dqrow       | False      |         20.367 |             50.469 | 46.392 |
+
+</details>
+
+But interestingly, if we use an exotic fpx scheme for quantization, we can afford lesser memory with an increase in the latency:
+
+<details>
+<summary>Collapse table</summary>
+
+|    | ckpt_id                      |   batch_size | fuse   | compile   | compile_vae   | quantization   | sparsify   |   model_memory |   inference_memory |   time |
+|---:|:-----------------------------|-------------:|:-------|:----------|:--------------|:---------------|:-----------|---------------:|-------------------:|-------:|
+|  0 | black-forest-labs/FLUX.1-dev |           16 | False  | True      | True          | fp6_e3m2       | False      |         17.591 |             49.938 | 61.649 |
```

Comment:
maybe we could optimize the time further

Reply:
Do you have any ideas on how?